### DISCUSSION TITLE
Can we enlarge the touch zones for going forward or back?

### DISCUSSION THREAD

**Discussion by saskir12** [maforget/ComicRackCE#11] [Ideas]
So I don't know if this is with all machines CR is installed but I find those hotzones where you can press for going forward or back a page too small. I am using a MS Surface device and I only have a small windows where I can click for going forward/back. Which is especially annoying as those are at the top and bottom which makes it uncomfortabel to hold. So is it possible to atleast make those larger or even give an option where you can maybe use the whole left or right part of the screen for this?

**Comment by 5h4k3** [comment]
If I correctly understand what your are looking for, you can resize pads in the "ComicRack.ini" that is in your CR installation folder.
You need to look at this line:

; Settings controlling gesture areas
 GestureAreaSize = 160
 ShowGestureHint = true
 
 These are not the default values, as I already modified them. They was too small for me, too. And I like to see them popping for a sec instead of trying to press randomly on the screen to find them.
